src/screens/CreatePost/index.js

```diff
@@ -717,7 +716,7 @@ const CreatePost = () => {
       animatedReminder.setValue(0);
     }
   }, [typeUser]);
-  console.log({typeUser}, 'heja');
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Nice work removing the console.log statement. It's a good practice not to leave debugging logs in production code.


<!-- This is an auto-generated comment by OSS CodeRabbit -->